R/coh_effectiveness.R

```diff
@@ -1,140 +1,99 @@
 #' @title Estimate Vaccine Effectiveness (VE)
 #'
-#' @description  This function provides methods for estimating VE. It relies
-#' on the implementation of the Kaplan-Meier estimator and the Cox model for
-#' proportional hazards in the package `{survival}`. Currently, the default
-#' method of the function is `HR` (Hazard Ratio). Thus, the VE = 1 - HR, where
-#' HR is calculated using the Cox model. The proportional hazards hypothesis is
-#' tested using the Schoenfeld test, and the resultant p-value is provided in
-#' the results. Log-log plots are also generated using the Kaplan-Meier
-#' survival estimator to provide a visual test for the proportional hazards
-#' hypothesis. The functions uses the default name of columns
-#' `outcome_status`, `time_to_event` and `vaccine_status`, and the  default
-#' status names `v` and `u`. However, custom names can be provided through the
-#' parameters `outcome_status_col`, `time_to_event_col`, `vacc_status_col`,
-#' `vaccinated_status` and `unvaccinated_status`.
-#' The return is a list with the call and the name of the method used for
-#' the estimation of VE (CI95%), the result of the performance test, and
-#' a suitable plot for the method.
-#' The object returned is compatible with the methods `summary` and `plot`.
+#' @description This function provides methods for estimating VE. It relies on
```

Comment:
I will keep it here because I defined the abbreviation in the title, but I'll be sure to use the abbreviations only where defined in the others functions